fineract-provider/build.gradle

```diff
@@ -298,6 +300,22 @@ if (project.hasProperty('security') && project.getProperty('security') == 'oauth
     }
 }
 
+modernizer {
+    includeTestClasses = false
+    failOnViolations = false
+
+    violationLogLevel="warn"
```

Comment:
Thanks for this contribution, I love it! Seeing that this only adds additional warn logs during the build, I fear everyone will just ignore this, and it won't really add any value... (This is why Checkstyle and SpotBugs are "enforced" now; if we don't do that, it's pretty much pointless.) I think what would be much more useful is if this was set to fail on violation, at violation log level error. I suspect that probably will cause some failure, but we should fix those...

Reply:
Ping @percyashu , will u like o address this so it can be merged by @vorburger ?

Reply:
I will make the changes and fix the violation.